internal/provider/custom_policy.go

```diff
@@ -165,13 +172,21 @@ func (r *customPolicyResource) Create(ctx context.Context, req resource.CreateRe
 		return
 	}
 
-	// Compute and validate the space
-	space, err := r.client.ComputeSpace(data.SpaceID)
-	if err != nil {
-		resp.Diagnostics.AddError("Invalid Configuration", err.Error())
-		return
+	var scopeMrn string
+	if !data.ScopeMrn.IsNull() {
+		scopeMrn = data.ScopeMrn.ValueString()
+	} else {
+		// Compute and validate the space
+		space, err := r.client.ComputeSpace(data.SpaceID)
+		if err != nil {
+			// resp.Diagnostics.AddError("Invalid Configuration", err.Error())
+			// return
```

Comment:
why did we comment this out? I think we still want to error is we don't have a scope Mrn 🤔 

if not, we should at least display a log saying what's happening here and remove this comments.

Reply:
i made scope mrn optional. But ill get this cleaned up